kuksa-val-server/src/VSSTypeSanitizer.cpp

```diff
@@ -165,8 +165,23 @@ void VssDatabase::checkAndSanitizeType(jsoncons::json &meta, jsoncons::json &val
         checkBoolType(val);
     }
     else if (dt == "string") { 
+      if (meta.contains("allowed") ) {
+        if (vss_3_0_supported) {
```

Comment:
This is one idea how to handle backward compatibility, have constants that specifies versions supported. Fills two purposes - one is to make it easy to remove support for certain versions in "own" forks, another is to make it easy to find version-dependent functionality.